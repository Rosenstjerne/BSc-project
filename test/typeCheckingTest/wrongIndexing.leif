var bool[] B;

B = new bool[5];
B[0] = true;
B[0] = true;
B[1] = false;
B[2] = true;
B[3] = false;
B[4] = true;

print(B[5 == 3]);
