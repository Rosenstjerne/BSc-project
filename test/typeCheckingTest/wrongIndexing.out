
Error in phase Type Checking, line 11:
    Type 'bool' can not be used to index into an array. Only allowed idexing type is 'int'
